ci/build/builder.go

```diff
@@ -100,6 +100,12 @@ func (build *Builder) WithAlpineBase() *Builder {
 	return &b
 }
 
+func (build *Builder) WithWolfiBase() *Builder {
+	b := *build
+	b.base = "wolfi"
+	return &b
+}
```

Comment:
If we could propagate something like this to the top-level of the `engine`, similar to how we do `GPUSupport` here: https://github.com/dagger/dagger/blob/5d0130c3c2650e0109f3daa0535ca74c5522f634/ci/engine.go#L41-L44

That way, building dagger based on wolfi could be just as simple as doing `dagger call --source=. engine with-base --base=wolfi container ...`

Reply:
SGTM

Reply:
Had a go at this once I figured out how the wiring worked. Following should work:

```
dagger call --source=. engine with-base --base=wolfi container export --path=./bin/engine.tar --forced-compression=Gzip
```

Note that I also removed the code to automatically set the base image to "ubuntu" when gpu support is requested -- now you need to explicitly set the base image to wolfi or ubuntu.